modules/developers-guide/pages/concepts/nodes-subnets.adoc

```diff
@@ -1,18 +1,18 @@
 = Nodes and sub-networks
-:keywords: Internet Computer,blockchain,protocol,replica,subnet,data center,canister,developer
+:keywords: Internet Computer,blockchain,protocol,replica,subnet,data center,smart contract,canister,developer
 :proglang: Motoko
 :platform: Internet Computer platform
 :IC: Internet Computer
 :company-id: DFINITY
 :sdk-short-name: DFINITY Canister SDK
 
-{IC} *sub-networks* provide physical hardware and resources—like CPU and memory—for performing software operations. Each sub-network consists of some number of individual machines—connected peer computers called *nodes*—that run the software components of the {IC} protocol. 
+{IC} *sub-networks* provide physical hardware and resources—like CPU and memory—for performing software operations. Each sub-network blockchain consists of some number of individual machines—connected peer computers called *nodes*—that run the software components of the {IC} protocol. 
 
-The {IC} software components that run on each node are called a *replica* because they replicate state and computation across all of the nodes in a sub-network.
+The {IC} software components that run on each node are called a *replica* because they replicate state and computation across all of the nodes in a sub-network blockchain.
```

Comment:
This sentence tells me that there is a separate blockchain for each sub-network. Is this true?